addon/components/pie-chart.js

```diff
@@ -83,7 +88,7 @@ const PieChartComponent = ChartComponent.extend(FloatingTooltipMixin,
         color: d.color,
         label: d.label,
         value: d.value,
-        percent: d3.round(100.0 * d.value / total, this.get('maxDecimalPlace'))
+        percent: 100.0*d.value / total
```

Comment:
Is there no limit to the amount of decimal precision that we're allowing?  It seems to me that having 0.000000000000000000000008 pie slice is not good.  Maybe I'm missing it, but are we filtering it out?